Source/WPF/MyMoney/Controls/TrendGraph.xaml.cs

```diff
@@ -320,29 +320,34 @@ private TrendGraphSeries AddSeries(DateTime start, DateTime end, Color color)
             IList<ChartValue> timeData = s.Values;
             s.BeginUpdate();
 
+            TrendValue emptyTrendValue = new TrendValue();
+            emptyTrendValue.Value = 0;
+            emptyTrendValue.UserData = null;
+
             DateTime last = start;
-            TrendValue lastv = null;
+            TrendValue lastv = emptyTrendValue;
             foreach (TrendValue v in this.data)
             {
-                // calculate balances using data from the start of the account \ list so the end balance is correct
-                if (v.Date <= end && v.Date >= start)
+                if (v.Date > end)

```

Comment:
This implies that the collection in this.data is presorted. Is that always true. If that's true maybe add a comment